packages/admin-panel/src/editor/Editor.js

```diff
@@ -26,19 +25,14 @@ export const Editor = ({ fields, recordData, onEditField, onSetFormFile }) => {
   }
 
   const onInputChange = async (inputKey, inputValue, editConfig = {}) => {
-    const { setFieldsOnChange, type } = editConfig;
-    let updatedValue = inputValue;
-    if (type === 'image' && inputValue) {
-      // If the input is a file, we need to convert the file to a base64 encoded image.
-      updatedValue = await createBase64Image(inputValue);
-    }
+    const { setFieldsOnChange } = editConfig;
```

Comment:
Uh... Idk what's going on here, 😅 @acdunham can you confirm these changes are good?

Reply:
They are all good :) Although... @WDuncker did you have any changes with `ui-components` to go with these FE changes? i think the `ui-components` package should now be handling the conversion to bas64 data image?